books/free-programming-books-subjects.md

```diff
@@ -225,6 +225,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 
 * [CI/CD for Monorepos: Effectively building, testing, and deploying code with monorepos](https://github.com/semaphoreci/book-monorepo-cicd) - Pablo Tomas Fernandez Zavalia, Marko Anastasov, SemaphoreCI (PDF, EPUB, Kindle)
 * [CI/CD with Docker and Kubernetes Book](https://github.com/semaphoreci/book-cicd-docker-kubernetes) - Marko Anastasov, Jérôme Petazzoni, Pablo Tom F. Zavalia, SemaphoreCI (PDF, EPUB, Kindle)
+* [Docker Cheat Sheet](https://github.com/wsargent/docker-cheat-sheet) - Will Sargent, et al.
```

Comment:
It's a cheat sheet.

Reply:
> it's a lot of cheat sheets!

Too much?

Reply:
can save pruning/glossing for later. Are any of them older? if so, maybe year would be useful?

Reply:
I'll check it :+1: again

Reply:
> It's a cheat sheet.

Resolved at 82457bc55591c61ff60afff83c302c5a4be7be87